_layouts/post.html

```diff
@@ -20,7 +20,7 @@ <h3 class="post-title" itemprop="name headline">{{ page.title | escape }}</h3>
         <div class="row">
             <div class="col-md-3 col-img">
                 {% assign image = site.data.images[page.image] %}
-                <img src="/img/{{ image.url }}" class="img-responsive postImg" />
+                <img src="/img/{{ image.url }}" class="img-responsive postImg" alt="post image" />
```

Comment:
This could be a bit more specific. E.g use
```Liquid
{% if image.name %}{{ image.name }}{% else %}post image{% endif %}
```

Reply:
yes, thanks. will do :+1: